### PR TITLE
Fix Deployment sidecar example

### PR DESCRIPTION
**Pull request by jboxman**
- https://github.com/openshift/openshift-docs/issues/22215

### CODE REVIEW

File: modules/ossm-automatic-sidecar-injection.adoc [hunk 1]
```diff
@@ -24,15 +24,20 @@ When deploying an application into the {ProductName} you must opt in to injectio
...
+    matchLabels:
+      app: sleep
   template:
     metadata:
-      annotations:
-        sidecar.istio.io/inject: "true"
       labels:
         app: sleep
+      annotations:
+        sidecar.istio.io/inject: "true"
```
Comment (tvieira) [inline modules/ossm-automatic-sidecar-injection.adoc:40]: This can be returned to its original position.

**Review by tvieira — CHANGES_REQUESTED** [review]

**Comment by tvieira** [comment]
tagging @neal-timpe for visibility.

**Review by neal-timpe — APPROVED** [review]
LGTM

**Review by sheriff-rh — APPROVED** [review]
LGTM!

**Pull request merged**